.github/workflows/build.yml

```diff
@@ -19,52 +19,85 @@ jobs:
             log-dir: "/var/log/opentelemetry/dotnet"
           - machine: macos-11
             log-dir: "/var/log/opentelemetry/dotnet"
+          - machine: actuated-arm64
+            log-dir: "/var/log/opentelemetry/dotnet"
     runs-on: ${{ matrix.machine }}
     steps:
-      - uses: actions/checkout@v4.1.1
+
+      - name: ARM64 Runner Info
+        if: ${{ matrix.machine == 'actuated-arm64' }}
+        run: cat /etc/os-release
+
+      - name: Checkout
+        uses: actions/checkout@v4.1.1
         with:
           fetch-depth: 0 # fetching all, needed to correctly calculate version
-      - uses: actions/setup-dotnet@v4.0.0
+
+      - name: Setup ARM64 Environment Variables
+        if: ${{ matrix.machine == 'actuated-arm64' }}
+        run: |
+          echo "DOTNET_INSTALL_DIR=~/.dotnet" >> $GITHUB_ENV
```

Comment:
Why this is not needed by the other architectures?

Reply:
`actions/setup-dotnet@v4.0.0` had somekind of issues running without root privileges. I guess other runners are configured differently.

Reply:
```
Run actions/setup-dotnet@v4.0.0
/home/runner/actions-runner/_work/_actions/actions/setup-dotnet/v4.0.0/externals/install-dotnet.sh --skip-non-versioned-files --runtime dotnet --channel LTS
mkdir: cannot create directory ‘/usr/share/dotnet’: Permission denied
```

Reply:
https://github.com/actions/setup-dotnet/issues/486